preset/data-analytics/docker-compose.yaml

```diff
@@ -35,6 +35,50 @@ services:
         https_proxy: ${https_proxy}
         no_proxy: ''
       context: .
+      labels:
+        docs: data_analytics
```

Comment:
note that the value here is the same value in `hook.py`, not the service name.

Reply:
got it, thanks!